sdk/identity/tests.yml

```diff
@@ -39,12 +39,6 @@ extends:
           PEM_CONTENTS: $(net-identity-spcert-pem)
           SNI_CONTENTS: $(net-identity-spcert-sni)
     EnvVars:
-      AZURE_IDENTITY_TEST_TENANTID: $(net-identity-tenantid)
-      AZURE_IDENTITY_TEST_USERNAME: $(net-identity-username)
-      AZURE_IDENTITY_TEST_PASSWORD: $(net-identity-password)
-      IDENTITY_SP_CLIENT_ID: $(net-identity-sp-clientid)
-      IDENTITY_SP_TENANT_ID: $(net-identity-sp-tenantid)
-      IDENTITY_SP_CLIENT_SECRET: $(net-identity-sp-clientsecret)
```

Comment:
What is replacing these?

Reply:
@christothes these are already defined in the cloud-specific subscription configuration secrets in keyvault (see [background docs](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/389/Configuring-Tests?anchor=configuring-subscriptions-and-cloud-environments)). Referencing secrets statically here means we can't set them dynamically per sovereign cloud, so I'm removing these references.

Reply:
Ahhh, because identity has no ARM template, these never get set by the script. Reverting, but we'll need to figure out a way to support this when we get around to sovereign cloud testing.